CODEOWNERS

```diff
@@ -333,6 +333,7 @@
 /scripts/gen_syscalls.py                  @andrewboie
 /scripts/process_gperf.py                 @andrewboie
 /scripts/gen_relocate_app.py              @wentongwu
+/scripts/receive_ctf.py                   @wentongwu
```

Comment:
I'd propose to change the name: It actually connects to serial, rather than doing anything with CTF. It just then _happens_ to be that what it receives is a CTF stream.

`receive_serial.py`

Oticon plans on contributing scripts to parse actual CTF streams (agnostic of IO delivery mechanism) for improved visualization in tools like TraceCompass and/or KernelShark. Hence to avoid confusion and name collision, this name change.

Reply:
@mped-oticon thanks, updated, I like your mentioned tools, please include me for review, thanks.